pcsx2/GS/GSState.cpp

```diff
@@ -1824,7 +1824,18 @@ inline bool GSState::TestDrawChanged()
 
 	return false;
 }
-
+u32 GSState::CalcMask(int exp, int max_exp)
+{
+	int amount = 9 + (max_exp - exp);
```

Comment:
Can be const.

Reply:
This is not ready for review, I guess I should draft it.